assets/js/components/notifications/UserInputSettings.js

```diff
@@ -70,9 +94,11 @@ export default function UserInputSettings( {
 			) }
 			format="large"
 			dismissExpires={ getTimeInSeconds( 'hour' ) * 3 }
-			ctaLink={ ctaLink }
-			ctaLabel={ __( 'Let’s go', 'google-site-kit' ) }
-			onCTAClick={ onCTAClick }
+			ctaComponent={
+				<Link href={ ctaLink } onClick={ onCTAClick }>
+					{ __( 'Let’s go', 'google-site-kit' ) }
```

Comment:
Looks like this text should be updated based on the [Figma design in the ACs of the issue](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=1620%3A48751&t=7LbhMja8OutnkU7p-0):

```suggestion
					{ __( 'Personalize your metrics', 'google-site-kit' ) }
```

Reply:
I updated the link text.